pom.xml

```
@@ -116,24 +116,49 @@
     </pluginRepository>
   </pluginRepositories>
 
-
-  <modules>
-    <module>kogito-build-parent</module>
-    <module>kogito-bom</module>
-    <module>kogito-test-utils</module>
-    <module>api</module>
-    <module>drools</module>
-    <module>jbpm</module>
-    <module>kogito-cloud-services</module>
-    <module>kogito-codegen-modules</module>
-    <module>kogito-springboot</module>
-    <module>kogito-maven-plugin</module>
-    <module>kogito-quarkus-extension</module>
-    <module>archetypes</module>
-    <module>grafana-api</module>
-    <module>addons</module>
-    <module>integration-tests</module>
-    <module>kogito-workitems</module>
-  </modules>
+  <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <module>kogito-build-parent</module>
+        <module>kogito-bom</module>
+        <module>kogito-test-utils</module>
+        <module>api</module>
+        <module>drools</module>
+        <module>jbpm</module>
+        <module>kogito-cloud-services</module>
+        <module>kogito-codegen-modules</module>
+        <module>kogito-springboot</module>
+        <module>kogito-maven-plugin</module>
+        <module>kogito-quarkus-extension</module>
+        <module>archetypes</module>
+        <module>grafana-api</module>
+        <module>addons</module>
+        <module>integration-tests</module>
+        <module>kogito-workitems</module>
+      </modules>
+    </profile>
+    <profile>
+      <id>prod</id>
+      <activation>
+        <property>
+          <name>prod</name>
```

i would use the same property we use in other assembllies
```suggestion
          <name>productized</name>
```